dds_cli/base.py

```diff
@@ -146,37 +145,12 @@ def __get_key(self, private: bool = False):
         """Get public key for project."""
         key_type = "private" if private else "public"
         # Get key from API
-        try:
-            response = requests.get(
-                DDSEndpoint.PROJ_PRIVATE if private else DDSEndpoint.PROJ_PUBLIC,
-                params={"project": self.project},
-                headers=self.token,
-                timeout=DDSEndpoint.TIMEOUT,
-            )
-        except requests.exceptions.RequestException as err:
-            LOG.fatal(str(err))
-            raise SystemExit(
-                "Failed to get cloud information"
-                + (
-                    ": The database seems to be down."
-                    if isinstance(err, requests.exceptions.ConnectionError)
-                    else "."
-                )
-            ) from err
-
-        if not response.ok:
-            message = "Failed getting key from DDS API"
-            if response.status_code == http.HTTPStatus.INTERNAL_SERVER_ERROR:
-                raise exceptions.ApiResponseError(message=f"{message}: {response.reason}")
-
-            raise exceptions.DDSCLIException(message=f"{message}: {response.json().get('message')}")
-
-        # Get key from response
-        try:
-            project_public = response.json()
-        except simplejson.JSONDecodeError as err:
-            LOG.fatal(str(err))
-            raise SystemExit from err
+        project_public = utils.request_get(
+            DDSEndpoint.PROJ_PRIVATE if private else DDSEndpoint.PROJ_PUBLIC,
+            params={"project": self.project},
+            headers=self.token,
+            error_message="Failed to get cloud information: The database seems to be down.",
```

Comment:
": The database seems to be down." is defined in the `request_get` function. Also, this old message is a bit confusing and a bit misrepresenting, so change that row to this: 
```suggestion
            error_message="Failed to project key",
```